base/celery-deployment.yaml

```diff
@@ -106,6 +106,12 @@ spec:
               value: '$(NEW_RELIC_MONITOR_MODE)'
             - name: FF_CLOUDWATCH_METRICS_ENABLED
               value: 'True'
+          lifecycle:
+          preStop:
+            exec:
+              command: ["/bin/bash"]
```

Comment:
Is `/bin/bash` in the API docker image?  If not, could you get away with using `/bin/sh`?

Reply:
it works when I checked it manually, I think it is okay.